### ISSUE TITLE
Subscription - "Request refund" button is missing from the subscription page

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#44899]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.4-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y 
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

**Issue found when executing PR** https://github.com/Expensify/App/pull/44138
## Action Performed:
Precondition: Have a Collest workspace and enabled subscription with a Gmail account
1. Navigate to https://staging.new.expensify.com/
2. Log in with the Gmail account
3. Navigate to https://staging.new.expensify.com/settings/subscription

## Expected Result:
"Request refund" button should be available

## Actual Result:
"Request refund" button is missing from the subscription page

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/38103576-34b1-4cc2-b1a3-b444f05dc4ab

![RRRBBB](https://github.com/Expensify/App/assets/78819774/684164b8-febe-488f-a250-6c5bf9bc08a6)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@RachCHopkins  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by RachCHopkins** [comment]
@lanitochka17 I'm not sure about this one. But I do see that request refund is available in Expensify Classic, so if there should be parity on this in New Expensify, then yes it is a bug:

![image](https://github.com/Expensify/App/assets/32854888/de8042c7-6b25-43db-b52e-9954681cd20e)

**Comment by trjExpensify** [comment]
We had a similar report with a retry payment function over here: https://github.com/Expensify/App/issues/44917

I [commented on that issue](https://github.com/Expensify/App/issues/44917#issuecomment-2213591679), but it would be good to figure out if this is personal/corporate karma related @MitchExpensify. 

Said differently, from the billing history above, $4.99 isn't a collect or control plan subscription.

**Comment by RachCHopkins** [comment]
Thanks @trjExpensify, personally I didn't think anyone could request refunds on these, but if the button is in OldDot it should be in NewDot. Unless it shouldn't be in OldDot, which is kind of what I am leaning towards.

**Comment by trjExpensify** [comment]
Yeah, I'm not 100%. Isn't that managed through Apple and IAP.

**Comment by RachCHopkins** [comment]
I only know from a Concierge Ops point of view:

1. Monthly individual subscriptions are paid in advance - when a user cancels, we turn off the SmartScans at the end of the period. Refund requests usually come in via Concierge and we process them via supportal
2. Karma subscriptions align with the billing period, and refund requests come in via Concierge, we process them directly in Stripe.

I've never known there to be a Request Refund button for either.

**Comment by trjExpensify** [comment]
That makes sense to me too. 👍

**Comment by RachCHopkins** [comment]
Ok so @trjExpensify does this mean it's actually an OldDot issue / Internal?

**Comment by trjExpensify** [comment]
Yeah, so a couple of things: 

1. We aren't showing anything to do with individual subscription billing in NewDot yet, that subset of customers aren't a focus right now. So this button shouldn't appear in NewDot for an individual subscription at this point regardless.

2. As for OldDot, I'm not sure we've concluded which subscription types we show the button for. I took a cursory look, and I found [this](https://github.com/Expensify/Auth/blob/b32438fc5a9ec75ba89461052cc52f189e0a6e8a/auth/lib/Account.cpp#L760-L771) for the isEligibleForRefund query. TYPE_MONTHLY & TYPE_YEARLY seem like they are collect/control related billingTypes when looking to corroborate that [here](https://github.com/Expensify/Auth/blob/b32438fc5a9ec75ba89461052cc52f189e0a6e8a/auth/lib/Account.cpp#L760-L771). But that might be only for the automated billing refund eligibility, and not simply allowing them to request a refund (and Conci handle the request manually).
